.gitignore

```diff
@@ -410,7 +410,6 @@ FodyWeavers.xsd
 .mtj.tmp/
 
 # Package Files #
-*.jar
```

Comment:
nit: we should not check-in jar files

Reply:
Addressed in commit [a29f16e](https://github.com/Azure/azsdk-java-rewrite-recipes/pull/14/commits/a29f16e6f730a6332908c520c0bd5244ffe4406c)